tests/journey/test_federal_senate_dataset.py

```diff
@@ -0,0 +1,34 @@
+import os
+from tempfile import gettempdir
+from unittest import TestCase, skipIf
+
+from serenata_toolbox.federal_senate.federal_senate_dataset \
+    import FederalSenateDataset
+
+
+class TestJourneyFederalSenateDataset(TestCase):
+    def setUp(self):
+        self.path = gettempdir()
+        self.subject = FederalSenateDataset(self.path)
+
+    @skipIf(os.environ.get('RUN_INTEGRATION_TESTS') != '1',
```

Comment:
Is sounds counter intuitive: if `RUN_INTEGRATION_TESTS` is not set, `0`, `False`, `''` (empty string) then we actually **run** the integration tests?